graph/src/data/store/mod.rs

```diff
@@ -846,6 +847,31 @@ impl Entity {
             }
         }
 
+        fn validate_fields<'a>(
```

Comment:
IMO this name is ambiguous, would prefer something like check_missing_fields or validate_missing_fields, since it's all it does and this way you can easily glean this information from the name

Reply:
thanks will make that change